src/RunQueryButtons.test.tsx

```diff
@@ -8,42 +8,42 @@ const getDefaultProps = (overrides?: Partial<RunQueryButtonsProps<DataQuery>>) =
   return {
     onRunQuery: jest.fn(),
     onCancelQuery: jest.fn(),
-    isQueryValid: jest.fn(),
+    enableRun: true,
     query: { refId: 'refId' },
     ...overrides,
   };
 };
 
 describe('RunQueryButtons', () => {
   it('disable the run button if the query is invalid', () => {
-    const props = getDefaultProps({ isQueryValid: jest.fn().mockReturnValue(false) });
+    const props = getDefaultProps({ enableRun: false});
```

Comment:
the test case description should be something like

'disable the run button if `enableRun` is false'